support/cas-server-support-redis-core/src/main/java/org/apereo/cas/redis/core/RedisObjectFactory.java

```diff
@@ -56,70 +61,74 @@
+            config.setTestOnReturn(props.isTestOnReturn());
+            config.setTestOnCreate(props.isTestOnCreate());
+            if (props.getMinEvictableIdleTimeMillis() > 0) {
+                config.setMinEvictableIdleTimeMillis(props.getMinEvictableIdleTimeMillis());
+            }
+            if (props.getNumTestsPerEvictionRun() > 0) {
+                config.setNumTestsPerEvictionRun(props.getNumTestsPerEvictionRun());
+            }
+            if (props.getSoftMinEvictableIdleTimeMillis() > 0) {
+                config.setSoftMinEvictableIdleTimeMillis(props.getSoftMinEvictableIdleTimeMillis());
+            }
+            poolConfig.poolConfig(config);
+        }
+        return poolConfig.build();
     }
 
-    private static LettucePoolingClientConfiguration redisPoolConfig(final BaseRedisProperties redis) {
-        val config = new GenericObjectPoolConfig();
-        val props = redis.getPool();
-        config.setMaxTotal(props.getMaxActive());
-        config.setMaxIdle(props.getMaxIdle());
-        config.setMinIdle(props.getMinIdle());
-        config.setMaxWaitMillis(props.getMaxWait());
-        config.setLifo(props.isLifo());
-        config.setFairness(props.isFairness());
-        config.setTestWhileIdle(props.isTestWhileIdle());
-        config.setTestOnBorrow(props.isTestOnBorrow());
-        config.setTestOnReturn(props.isTestOnReturn());
-        config.setTestOnCreate(props.isTestOnCreate());
-
-        if (props.getMinEvictableIdleTimeMillis() > 0) {
-            config.setMinEvictableIdleTimeMillis(props.getMinEvictableIdleTimeMillis());
-        }
-        if (props.getNumTestsPerEvictionRun() > 0) {
-            config.setNumTestsPerEvictionRun(props.getNumTestsPerEvictionRun());
-        }
-        if (props.getSoftMinEvictableIdleTimeMillis() > 0) {
-            config.setSoftMinEvictableIdleTimeMillis(props.getSoftMinEvictableIdleTimeMillis());
+    private static RedisStandaloneConfiguration getStandaloneConfig(final BaseRedisProperties redis) {
+        LOGGER.trace("Setting Redis standalone configuration on host '{}' and port '{}'", redis.getHost(),
+                redis.getPort());
+        val standaloneConfig = new RedisStandaloneConfiguration(redis.getHost(), redis.getPort());
+        standaloneConfig.setDatabase(redis.getDatabase());
+        if (StringUtils.hasText(redis.getPassword())) {
+            standaloneConfig.setPassword(RedisPassword.of(redis.getPassword()));
         }
-        return LettucePoolingClientConfiguration.builder()
-            .poolConfig(config)
-            .build();
+        return standaloneConfig;
     }
 
-    private static RedisSentinelConfiguration potentiallyGetSentinelConfig(final BaseRedisProperties redis) {
+    private static RedisSentinelConfiguration getSentinelConfig(final BaseRedisProperties redis) {
+        LOGGER.trace("Setting Redis with Sentinel configuration on master '{}'", redis.getSentinel().getMaster());
```

Comment:
To keep things consistent, use `[{}]` rather than `{}`.

Reply:
PS Later on, I think we should be able to add checkstyle rules to enforce this sort of thing and make it easier for folks to do the work upfront. It's no big deal of course if you wish to keep things are they are.

Reply:
Yes sorry it's a more consistent way from the whole project.
And yes a checkstyle would be requested to avoid a such review.